server/app/src/main/java/com/ams/app/controller/SubjectController.java

```diff
@@ -0,0 +1,50 @@
+package com.ams.app.controller;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.ams.app.model.Subject;
+import com.ams.app.service.SubjectService;
+
+@RestController
+@RequestMapping("/subject")
+public class SubjectController {
+
+    @Autowired
+    SubjectService subjectService;
```

Comment:
It's good to use Access Modifiers like private, public.. In this case, subjectService can be private

Reply:
Modified and used private for all.